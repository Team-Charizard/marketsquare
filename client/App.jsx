import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import Login from './components/login';
import MainContainer from './components/MainContainer';
import LandingPage from './components/LandingPage';

const App = () => {
  return (
    <div>
      <Router>
        <MainContainer />
        <Login />
        <Route exact path='/' component={Login} />
        <Route exact path='/LandingPage' component={LandingPage} />
      </Router>
    </div>
  );
};
export default App;
