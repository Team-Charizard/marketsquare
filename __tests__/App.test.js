import React from 'react';
import { shallow } from 'enzyme';
import App from '../client/App';

it('Snapshot Test', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});
