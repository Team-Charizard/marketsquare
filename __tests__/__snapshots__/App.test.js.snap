// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App unit tests Snapshot Test 1`] = `
<div>
  <Connect(MainContainer) />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/LandingPage"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(CreateUser)",
          "type": [Function],
        }
      }
      exact={true}
      path="/CreateUser"
    />
  </Switch>
</div>
`;
